### DISCUSSION TITLE
Consider OSGI compatibility

### DISCUSSION THREAD

**Discussion by dwulfert** [instancio/instancio#472] [Ideas]
First of all, thank you for your effort. This project seems to be exactly what I was looking for. Unfortunately, I need to integrate the dependency into an OSGI project, which can be quite complicated without integrated bundle information. To make it OSGI compatible, I suggest integrating the maven-bundle-plugin as the only necessary plugin.

**Comment by armandino** [comment]
@dwulfert thank you for the suggestion. I created an issue (#473) from this post and will try to add OSGi support in the next release.
